BundleTransport/app/src/main/java/net/discdd/bundletransport/TransportToBundleServerManager.java

```diff
@@ -3,6 +3,11 @@
 import static java.util.logging.Level.INFO;
 import static java.util.logging.Level.SEVERE;
 
+import android.content.Context;
```

Comment:
why were these imports added? you probably want to run the optimized imports helper.